tests/circuit.test.ts

```diff
@@ -75,6 +75,8 @@ describe("Circuit test", () => {
     await circuit.calculateWitness(input);
     const p = (await circuit.genProof(input)) as ZK_PROOF;
 
+    fs.writeFileSync("./tmp.json", JSON.stringify(p))
```

Comment:
@bajpai244 	once the proof is generated, we should delete this file. Also any reasons to commit this `tmp.json`?

Reply:
Yeah, correct, I just pushed a change resolving that.